build-tools/libzip-windows/libzip-windows.projitems

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <_LibZipTarget Include="host-mxe-Win64" Condition="$(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+      <CMake>$(AndroidMxeFullPath)\bin\x86_64-w64-mingw32.static-cmake</CMake>
+      <CMakeFlags></CMakeFlags>
+      <OutputLibrary>libzip.dll</OutputLibrary>
+      <OutputLibraryPath>lib/libzip.dll</OutputLibraryPath>
```

Comment:
That's what it *was*. That said, I'm not certain that a Windows path will *work*.

> in other humorous xbuild news, `\` within item metadata ​_is not_​ replaced with `/` in file system paths when using `xbuild`.
> 
> e.g. given `%(_MonoCrossRuntime.InstallPath)%(_MonoCrossRuntime.CrossMonoName)`, where `%(_MonoCrossRuntime.InstallPath)` is `bin\` (note backslash), the generated file in a `<Touch/>` and `<Copy/>` is `bin\cross-whatever`, ​_not_​ my expected `bin/cross-whatever`